com.github.leeroymanea.synology-note-station.yaml

```diff
@@ -0,0 +1,40 @@
+# copied/inspired from https://github.com/flathub/com.synology.SynologyDrive/blob/master/com.synology.SynologyDrive.yaml
+id: com.github.leeroymanea.synology-note-station
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+arch:
+  - x86_64
+tags:
+  - proprietary
+command: /app/opt/synology-note-station/launch.sh
+finish-args:
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+modules:
+  - name: synology-note-station
+    buildsystem: simple
+    sources:
+      - type: file
+        sha256: a4c763bddda21bd211b0c8afdcff7e72a3edddc2392470fec8ce8123a0531751
+        url: "https://global.synologydownload.com/download/Utility/NoteStationClient/2.2.4-703/Ubuntu/x86_64/synology-note-station-client-2.2.4-703-linux-x64.deb"        
+      - type: file
+        path: com.github.leeroymanea.synology-note-station.metainfo.xml
+    build-commands:
+       # unzip folder
+      - ar x synology-note-station-client-2.2.4-703-linux-x64.deb
+      - tar -xf data.tar.xz
+      # move to app location
+      - mv usr/share /app/share
+      - mv opt /app/opt
+      # rename some files
+      - mv /app/share/applications/synology-note-station-client.desktop /app/share/applications/com.github.leeroymanea.synology-note-station.desktop
+      - find /app/share/icons/hicolor -type f -execdir sh -c 'mv "$0" com.github.leeroymanea.synology-note-station.png' {} \;
+      # update desktop file
+      - desktop-file-edit --remove-key=Version /app/share/applications/com.github.leeroymanea.synology-note-station.desktop
```

Comment:
Why remove version?

Reply:
someone told me the last time i tried to push an app to flathub

Reply:
i will remove this line

Reply:
the "original" version value is invalid  
```
error: value "2.0" for key "Version" in group "Desktop Entry" is not a known version
```